README.rst

```diff
@@ -87,107 +87,162 @@ Fletch
+   :alt: KWIVER CMake Configuration Step 3
 
 Compiling
 ---------
 
-Once your CMake command has completed and generated the build files,
+Once your CMake generation has completed and created the build files,
 compile in the standard way for your build environment.  On Linux
 this is typically running ``make``.
 
+There is also a build target, INSTALL. This target will build all code, then create an install directory inside the build directory.
+This install folder will be populated with all binaries, libraries, headers, and other files you will need to develop your application with kwiver. 
+MSVC users note, this install directory is for a single build configuration and their will not be configuration 
+named directories in this directory structure. (i.e. no /bin/release, only /bin)
+
 
 Running KWIVER
 ==============
 
-Once you've built KWIVER, you'll want to test that it's working on
-your system.  From a command prompt execute the following command::
+Once you've built KWIVER, you'll want to test that it's working on your system.
+From a command prompt execute the following command::
 
-	source </path/to/kwiver/build>/install/setup_KWIVER.sh
+  # via a bash shell
+  source </path/to/kwiver/build>/setup_KWIVER.sh
+  #
+  # via a windows cmd prompt
+  </path/to/kwiver/build>/setup_KWIVER.bat
 
 Where `</path/to/kwiver/build>` is the actual path of your KWIVER
 CMake build directory.
 
 This will set up your PATH, PYTHONPATH and other environment variables
-to allow KWIVER to work conveniently.
+to allow KWIVER to work conveniently within in the shell/cmd window.
 
+You can run this simple pipeline to ensure your system is configured properly::
 
-Vital
-=====
+  # via a bash shell
+  </path/to/kwiver/build>/bin/pipeline_runner -p </path/to/kwiver/source>/sprokit/pipelines/number_flow.pipe
+  #
+  # on windows, you will need to also be in the configuration folder
+  </path/to/kwiver/build>/bin/release/pipeline_runner -p </path/to/kwiver/source>/sprokit/pipelines/number_flow.pipe
 
-Vital is an open source C++ collection of libraries and tools that
-supply basic types and services to the Kitware KWIVER imagery tool
-kit.
+This will generate a 'numbers.txt' file in the current directory (/bin or /bin/release).
 
-Overview of Directories
------------------------
+KWIVER Users
+============
+
+Here are some applications using kwiver that serve as an example of how to leverage kwiver for a specific application
+
+======= ===================================================================
+MAP-Tk_  A collection of libraries and tools for making measurements from aerial video
+VIAME_   A computer vision library designed to integrate several image and video processing algorithms together in a common              distributed processing framework, majorly targeting marine species analytics
+======= ===================================================================
+
+Code Structure and Provided Functionality
```

Comment:
I was thinking about removing this section altogether or moving it different doc.   There are several directories here containing experimental content or implementation details.  I'm not sure if we want to go in this much depth in the README.  What do you think?  I'm on the fence.